docs/deployment/rds-s3/auto-rds-s3-setup.py

```diff
@@ -645,12 +649,12 @@ def footer():
     help=f"Unique identifier for the RDS database instance. Default is set to {DB_INSTANCE_NAME_DEFAULT}",
     required=False
 )
-DB_NAME_DEFAULT = "kubeflow"
+DB_NAME_DEFAULT = "ml_metadata_db"
```

Comment:
[nit] can we keep the default same as the vanilla : `metadb`

Reply:
Sure!

Reply:
Done